mobile-center-link-scripts/src/ios/MobileCenterConfig.js

```diff
@@ -40,7 +40,15 @@ function addConfigToProject(file) {
     return new Promise(function (resolve, reject) {
         debug(`Trying to add ${file} to XCode project`);
 
-        var globString = `**/${(pjson && pjson.name ? pjson.name : '*')}.xcodeproj/project.pbxproj`
+        var globString;
+        if (pjson && pjson.name) {
+            globString = `**/${(pjson.name.replace(/-/g, ''))}`;
+        }
+        else {
+            globString = `ios/*`;
+        }
+        globString += '.xcodeproj/project.pbxproj';
+
         var projectPaths = glob.sync(globString, { ignore: 'node_modules/**' });
```

Comment:
IMHO, we should look at what React-Native looks for xcode project - this should mimic that.